packages/e2e-tests/config/setup-test-framework.js

```diff
@@ -193,19 +193,23 @@ async function runAxeTestsForBlockEditor() {
 			'aria-hidden-focus',
 			'aria-input-field-name',
 			'aria-valid-attr-value',
-			'button-name',
-			'color-contrast',
-			'dlitem',
+			// 'button-name',
+			// 'color-contrast',
+			// 'dlitem',
 			'duplicate-id',
-			'label',
-			'landmark-one-main',
+			// 'label',
+			// 'landmark-one-main',
 			'link-name',
-			'listitem',
-			'region',
+			// 'listitem',
+			// 'region',
```

Comment:
I might have missed it in the discussion.  Was there a reason we are commenting out a handful of the disabled rules?